core/internal.d.ts

```diff
@@ -28,1043 +28,1039 @@
 /// <reference no-default-lib="true" />
 /// <reference lib="esnext" />
 
-declare namespace __bootstrap {
-  /**
-   * Primordials are a way to safely use globals without fear of global mutation
-   * Generally, this means removing `this` parameter usage and instead using
-   * a regular parameter:
-   *
-   * @example
-   *
-   * ```js
-   * 'thing'.startsWith('hello');
-   * ```
-   *
-   * becomes
-   *
-   * ```js
-   * primordials.StringPrototypeStartsWith('thing', 'hello')
-   * ```
-   */
-  declare namespace primordials {
-    type UncurryThis<T extends (this: unknown, ...args: unknown[]) => unknown> =
-      (self: ThisParameterType<T>, ...args: Parameters<T>) => ReturnType<T>;
-    type UncurryThisStaticApply<
-      T extends (this: unknown, ...args: unknown[]) => unknown,
-    > = (self: ThisParameterType<T>, args: Parameters<T>) => ReturnType<T>;
-    type StaticApply<T extends (this: unknown, ...args: unknown[]) => unknown> =
-      (args: Parameters<T>) => ReturnType<T>;
+/**
+ * Primordials are a way to safely use globals without fear of global mutation
+ * Generally, this means removing `this` parameter usage and instead using
+ * a regular parameter:
+ *
+ * @example
+ *
+ * ```js
+ * 'thing'.startsWith('hello');
+ * ```
+ *
+ * becomes
+ *
+ * ```js
+ * primordials.StringPrototypeStartsWith('thing', 'hello')
+ * ```
+ */
+declare namespace primordials {
```

Comment:
This isn't going to work. If you want to make these files still somewhat useful, you'll need to either use `module "internal:core/..." { export namespace primordials { ... } }` or something similar